.spelling

```diff
@@ -148,6 +150,8 @@ ControlZ
 CoreDNS
 coreos
 Costin
+CPU
```

Comment:
Remove. Again surround with backticks in your doc.

Reply:
I see that the word `CPU` is not listed alphabetically in `.spelling` so I move it up.

Reply:
Lulz, @costinm is in the spelling file :)